src/nodes.coffee

```diff
@@ -1532,6 +1532,9 @@ exports.Param = class Param extends Base
         obj = obj.variable
       # * assignments within destructured parameters `{foo:bar}`
       if obj instanceof Assign
+        # ... possibly with a default value
+        if obj.value instanceof Assign
```

Comment:
You could argue it's **very** weird the first `=` has `?` semantics, when the second one doesn't (I guess it's why @satyr made `?` work in coco, huh.)